components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.common/src/main/java/org/wso2/carbon/identity/api/server/idp/common/Constants.java

```diff
@@ -93,9 +93,31 @@
      */
     public enum ErrorMessage {
 
+        // Client errors starting from 600xx.
+        ERROR_CODE_IDP_NOT_FOUND("60002", "Resource not found.", "Unable to find a resource matching the provided " +
+                "identity provider identifier %s."),
         ERROR_CODE_ERROR_LISTING_IDPS("60021",
                 "Unable to list existing identity providers.",
                 "Server encountered an error while listing the identity providers."),
+        ERROR_CODE_AUTHENTICATOR_NOT_FOUND_FOR_IDP("60022",
+                "Resource not found.",
+                "Unable to find federated authenticator with identifier %s."),
+        ERROR_CODE_CONNECTOR_NOT_FOUND_FOR_IDP("60023", "Resource not found.", "Unable to find outbound provisioning " +
+                "connector identifier %s."),
+        ERROR_CODE_INVALID_LOCAL_CLAIM_ID("60024",
+                "Invalid claim uri.",
+                "Invalid claim uri %s provided in claim config."),
+        ERROR_CODE_INVALID_INPUT("60025", "Invalid input.", "One of the given inputs is invalid."),
+        ERROR_CODE_INVALID_SAML_METADATA("60026", "Invalid SAML metadata.", "SAML metadata is invalid/empty."),
+        ERROR_CODE_INVALID_DEFAULT_AUTHENTICATOR("60027", "Invalid default authenticator ID",
+                "Provided value for the default authenticator ID is invalid."),
+        ERROR_CODE_INVALID_DEFAULT_OUTBOUND_CONNECTOR("60027", "Invalid default outbound connector ID",
```

Comment:
Same error code repeated 60027

Reply:
With the commit, I have rearranged the order of the error codes and I have not changed the content. This was an existing issue. I think we cannot change the error codes now.

Reply:
Let's change this. It would be still fine than proceeding this way

Reply:
Fixed: https://github.com/wso2/identity-api-server/pull/206/commits/b845597e3a3eba162465ff7648b6cee3fccdf71d